### PR TITLE
Adding LDAP-Auth & Docs

### LINKED ISSUE #1173: Requesting LDAP-auth
# Feature request

**Describe the solution you'd like**
Add linuxserver/ldap-auth so that users can more easily add ldap authentication to nginx

### PR DESCRIPTION
**Pull request by angrycuban13**
# Pull request

**Purpose**
Closes #1173

### CODE REVIEW

File: compose/.apps/ldapauth/ldapauth.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      - TZ=${TZ}
+    logging:
+      driver: json-file
+      options:
+        max-file: ${DOCKERLOGGING_MAXFILE}
+        max-size: ${DOCKERLOGGING_MAXSIZE}
+    restart: unless-stopped
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - ${DOCKERSHAREDDIR}:/shared
```
Comment (nemchik) [inline compose/.apps/ldapauth/ldapauth.yml:17]: don't include shared with new apps

**Review by nemchik — CHANGES_REQUESTED** [review]
Some files missing a blank newline as the last line also.

File: compose/.apps/ldapauth/ldapauth.yml (hunk 1 above)
Comment (angrycuban13) [reply compose/.apps/ldapauth/ldapauth.yml]: e82356c

**Comment by nemchik** [comment]
Some failing checks here:

- image issue? not sure what's up
- editorconfig: probably trailing newlines missing
- navigation link

**Comment by angrycuban13** [comment]
Yeah looks like my CTRL+F fucked some stuff up. Should be fixed now.

**Review by nemchik — APPROVED** [review]

**Pull request merged**